deploy/sample-backup-configs/sample_config_build_recipes.ini

```diff
@@ -42,7 +42,23 @@ instancetype=z1d.2xlarge
 deploytriplet=None
 
 
-# Single-core, Rocket-based recipes
+# Single-core, Ariane-based recipes
+[firesim-ariane-singlecore-nic-l2-llc4mb-ddr3]
+DESIGN=FireSim
+TARGET_CONFIG=WithNIC_DDR3FRFCFSLLC4MB_FireSimArianeConfig
+PLATFORM_CONFIG=F90MHz_BaseF1Config
+instancetype=z1d.2xlarge
+deploytriplet=None
+
```

Comment:
I would prefer to include only the no-nic option here, to reduce sample recipe bloat.